src/contracts/interfaces/IEigenPodManager.sol

```diff
@@ -39,34 +56,58 @@ interface IEigenPodManager is IPausable {
      * @param sharesDelta is the change in podOwner's beaconChainETHStrategy shares
      * @dev Callable only by the podOwner's EigenPod contract.
      */
-    function recordBeaconChainETHBalanceUpdate(address podOwner, uint256 beaconChainETHStrategyIndex, int256 sharesDelta) external;
-    
+    function recordBeaconChainETHBalanceUpdate(address podOwner, int256 sharesDelta) external;
+
+
+    /**
+     * @notice queues a withdrawal beacon chain ETH from EigenLayer on behalf of the owner of an EigenPod.
+     * @param amountWei is the amount of ETH to withdraw
+     * @param undelegateIfPossible is whether or not to undelegate the shares if possible
+     * @param alsoWithdraw is whether or not to also withdraw the ETH from the beacon chain vs. redelegating to a new operator in EL
+     */
+    function queueWithdrawal(uint256 amountWei, bool undelegateIfPossible, bool alsoWithdraw) external returns(bytes32);
+
+    /**
+     * @notice forces a withdrawal of the podOwner's beaconChainETHStrategy shares
+     * @param podOwner is the pod owner whose shares are to be removed
+     */
+    function forceWithdrawal(address podOwner) external returns (bytes32);
+
+
+    /** 
+     * @notice slashes a pending queued withdrawal of the podOwner's beaconChainETHStrategy shares
+     * @param slashedFundsRecipient is the address to receive the slashed funds
+     * @param queuedWithdrawal is the queued withdrawal to be slashed
+     */
+    function slashQueuedWithdrawal(address slashedFundsRecipient, BeaconChainQueuedWithdrawal memory queuedWithdrawal) external;
+
     /**
-     * @notice Withdraws ETH from an EigenPod. The ETH must have first been withdrawn from the beacon chain.
-     * @param podOwner The owner of the pod whose balance must be withdrawn.
-     * @param recipient The recipient of the withdrawn ETH.
-     * @param amount The amount of ETH to withdraw.
-     * @dev Callable only by the StrategyManager contract.
+     * @notice slashes shares of the podOwner and sends them to the slashedFundsRecipient
+     * @param slashedPodOwner is the address of the pod owner whose shares are to be slashed
+     * @param slashedFundsRecipient is the address to receive the slashed funds
+     * @param shareAmount is the amount of shares to be slashed     
      */
-    function withdrawRestakedBeaconChainETH(address podOwner, address recipient, uint256 amount) external;
+    function slashShares(address slashedPodOwner, address slashedFundsRecipient, uint256 shareAmount) external;
 
+
+    /**
+     * @notice Completes an existing queuedWithdrawal either by sending the ETH to the recipient or allowing the podOwner to re-delegate it
+     * @param queuedWithdrawal is the queued withdrawal to be completed
+     * @param middlewareTimesIndex is the index in the operator that the staker who triggered the withdrawal was delegated to's middleware times array
+     * @dev Callable only by the podOwner's EigenPod contract.
```

Comment:
seems like potentially some very odd routing that the EigenPod is calling this rather than the user themselves?
I'll look into it more tho

Reply:
oh I didn't touch the documentation really, thats all autocomplete